x-pack/plugins/security_solution/cypress/integration/detection_alerts/acknowledged.spec.ts

```diff
@@ -63,3 +64,41 @@ describe.skip('Marking alerts as acknowledged', () => {
       });
   });
 });
+
+describe.skip('Marking alerts as acknowledged with read only role', () => {
```

Comment:
Will these be checked in skipped?

Reply:
Well i don't want them to be, but with the cypress failures right now, I might follow @FrankHassanabad's advice and check them in skipped. It's the same problem thats occurring some other places where they pass locally and fail on CI. I'll check the release dates on these versions and see if we've got time to wait and work on a fix for those first